main.py

```diff
@@ -41,11 +41,12 @@ def main():
             print("\n1. Add Grade")
             print("2. Edit Grade")
             print("3. View Grades")
-            print("4. Logout")
+            print("4. Sort Grades")
+            print("5. Logout")
```

Comment:
To prevent future changes in the menu options for logout. Please change the option for logout from 5 to 'x'. That means that user can log out by inputting x.